decidim-core/app/mailers/concerns/decidim/multitenant_asset_host.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A module to be included in mailers that changes the default behaviour so
+  # the emails are rendered in the user's locale instead of the default one.
+  module MultitenantAssetHost
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :set_asset_host
+
+      # set_asset_host allows to redefine the host asset if not specified in application configuration
+      def set_asset_host
+        return if Rails.application.config.action_mailer.asset_host.present?
+
+        self.asset_host = ->(_mail) { "#{use_https? ? "https" : "http"}://#{@organization.host}" }
```

Comment:
I can't see this documented in https://guides.rubyonrails.org/action_mailer_basics.html. Can you share in module's doc what are we hacking here? (where is this attribute defined, etc.)

Reply:
Assets host is set on startup, therefore I used a lambda to execute it only when a mail is sent. I could not manage to use a lambda without a parameter, so I used an optional one.

I added documentation to the module.